ec2-spot-deep-learning-training/ec2-spot-deep-learning.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -x
+
+echo "Hello World from EC2 Spot Team..."
+#Global Defaults
+DEFAULT_REGION=us-east-1
```

Comment:
You did the change before to find out the region, any reason why you are not making it here ?